libr/include/r_cons.h

```diff
@@ -736,6 +739,9 @@ R_API void r_cons_flush(void);
 R_API void r_cons_last(void);
 R_API int r_cons_less_str(const char *str, const char *exitkeys);
 R_API void r_cons_less(void);
+R_API int r_cons_next_match(int from, RList **mla, int lcount);
```

Comment:
use R_IPI, those functions should not be exported. they  are just interenal to rcons

Reply:
and  use different naming, because r_cons implies public

Reply:
OK, where should I declare these functions then?

Reply:
just change this to R_IPI for now, but we can add a private.h inside libr/cons with those definitions and include "./private.h" where needed.